app-misc/pax-utils/pax-utils-1.3.3.ebuild

```diff
@@ -68,7 +68,7 @@ src_test() {
 }
 
 src_install() {
-	_emake DESTDIR="${D}" PKGDOCDIR='$(DOCDIR)'/${PF} install
+	_emake DESTDIR="${ED}" PKGDOCDIR='$(DOCDIR)'/${PF} install
```

Comment:
Please drop this change, 1.3.3 is not affected. I just installed it on my prefix without this change and all files get installed into the right places already.